src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -239,3 +468,22 @@ def delete(self) -> None:
             datasets_api_v1.delete_dataset(client=self._client, id=self.id)
         except Exception as e:
             raise RuntimeError(f"Failed while deleting the `FeedbackDataset` from Argilla with exception: {e}") from e
+
+    @classmethod
+    def __copy_from(cls, dataset: "RemoteFeedbackDataset") -> "RemoteFeedbackDataset":
```

Comment:
Since this logic is not part of the API definition, I would avoid exposing it as public. This method just creates a dataset copy using as source the passed as argument. We can call it `__create_copy`